apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.scss

```diff
@@ -0,0 +1,11 @@
+.bg-negative {
+    color: red;
+}
+
+.bg-ok {
+    color: orange;
+}
+
+.bg-positive {
+    color: green;
+}
```

Comment:
Please use appropriate naming. It should also contain the word `docs` so the user is not confused where this class is coming from

Reply:
fixed.